R/ranges_helpers.R

```diff
@@ -415,11 +417,12 @@ overlapsToCoverage <- function(gr, reads, keep.names = TRUE, type = "any") {
 #' Get coverage per group
 #'
 #' It tiles each GRangesList group, and finds hits per position
+#'
+#' This is a safer speedup of coverageByTranscript in GenomicFeatures.
```

Comment:
This is a safer speedup of coverageByTranscript from GenomicFeatures. or GenomicFeatures::